src/terrain/clients/analyses.clj

```diff
@@ -0,0 +1,40 @@
+(ns terrain.clients.analyses
+  (:require [terrain.util.config :refer [analyses-base-uri]]
+            [clj-http.client :as http]
+            [cemerick.url :refer [url]]
+            [terrain.auth.user-attributes :refer [current-user]]
+            [slingshot.slingshot :refer [throw+]]))
+
+
+(defn analyses-url
+  ([components]
+   (analyses-url components {}))
+  ([components query]
+   (-> (apply url (cons (analyses-base-uri) components))
+       (assoc :query (assoc query :user (:username current-user)))
+       (str))))
+
+(defn get-badge
+  [id]
+  (:body (http/get (analyses-url ["badges" id]) {:as :json})))
+
+(defn delete-badge
+  [id]
+  (http/delete (analyses-url ["badges" id]) {:as :json}) {:id id}
```

Comment:
I could be wrong, but I think that the `{:id id}` form on the end of this line is extraneous.

Reply:
Yeah, it's a hold-over from similar code in the analyses service. I'll just return the body of of the deletion response.

Reply:
This should now be dealt with.

Reply:
Actually, I was wrong. It a hold over from the parse-response parameters. I've nuked it.